Classes/Private/Manager/Video/OCTVideoView.m

```diff
@@ -47,8 +48,16 @@ - (void)setImage:(CIImage *)image
 - (void)drawRect:(CGRect)rect
 {
     if (self.image) {
-        CGFloat scale = self.window.screen.scale;
-        CGRect destRect = CGRectApplyAffineTransform(self.bounds, CGAffineTransformMakeScale(scale, scale));
+
+        glClearColor(0, 0.0, 0.0, 1.0);
+        glClear(GL_COLOR_BUFFER_BIT);
```

Comment:
Hmm yea I could probably optimize this a bit. Basically check if the latest `self.image.extent` has changed, and if it did, call `glClear`